apps/docs/stories/components/ModularCheckout/Sub-components/ApplePay/Docs.mdx

```diff
@@ -30,3 +30,40 @@ Renders an Apple Pay button for eligible devices and orchestrates the Apple Pay
 - `applePayCancelled`
 - `applePayError`
 
+### Event detail shapes
+
+- `applePayCompleted`
+  - Detail: `{ success: boolean; token?: IApplePayToken; paymentMethodId?: string; error?: any }`
+- `applePayError`
+  - Detail: `{ error: string; code: string }`
+  - `code` values: `CONFIG_ERROR`, `NOT_SUPPORTED`, `NOT_AVAILABLE`, `INITIALIZATION_ERROR`, `PAYMENT_FAILED`
```

Comment:
Based on the changes to `apps/docs/stories/components/ModularCheckout/Docs.mdx`, I think these are supposed to be `APPLE_PAY_CONFIG_ERROR`, `APPLE_PAY_NOT_SUPPORTED`, etc